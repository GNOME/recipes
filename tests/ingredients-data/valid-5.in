a apple
an apple
one apple
two apples
three apples
four apples
five apples
six apples
seven apples
eight apples
nine apples
ten apples
eleven apples
twelve apples
a dozen apples
