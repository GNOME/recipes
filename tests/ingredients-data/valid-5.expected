AMOUNT 1/1
UNIT (null)
NAME Apple

AMOUNT 1/1
UNIT (null)
NAME Apple

AMOUNT 1/1
UNIT (null)
NAME Apple

AMOUNT 2/1
UNIT (null)
NAME Apple

AMOUNT 3/1
UNIT (null)
NAME Apple

AMOUNT 4/1
UNIT (null)
NAME Apple

AMOUNT 5/1
UNIT (null)
NAME Apple

AMOUNT 6/1
UNIT (null)
NAME Apple

AMOUNT 7/1
UNIT (null)
NAME Apple

AMOUNT 8/1
UNIT (null)
NAME Apple

AMOUNT 9/1
UNIT (null)
NAME Apple

AMOUNT 10/1
UNIT (null)
NAME Apple

AMOUNT 11/1
UNIT (null)
NAME Apple

AMOUNT 12/1
UNIT (null)
NAME Apple

AMOUNT 12/1
UNIT (null)
NAME Apple

